src/CSS.cpp

```diff
@@ -58,15 +60,31 @@ namespace CSS
         GError* err = nullptr;
         for (auto& dir : locations)
         {
-            std::string file = dir + "/style.css";
-
-            if (!std::ifstream(file).is_open())
+            std::string file = dir + "/style.scss";
+            std::ifstream scss(file);
+            if (std::ifstream(file).is_open())
             {
-                LOG("Info: No CSS found in " << dir);
-                continue;
+                Sass_File_Context* ctx =  sass_make_file_context(file.c_str());
+                Sass_Context* ctxout = sass_file_context_get_context(ctx);
+                sass_compile_file_context(ctx);
+                std::string data = sass_context_get_output_string(ctxout);
+                gtk_css_provider_load_from_data(sProvider, data.c_str(), data.length(), &err);
+                sass_delete_file_context(ctx);
```

Comment:
There needs to be some sort of error handling here (preferably outputting the error string and then trying to load other locations). If the compilation step fails, we would simply fail to load the css without falling back to other options here.

Reply:
fixed

Reply:
the exact error messages and logging behaviour is maybe something to be discussed though